docs/Using-Fleet/REST-API.md

```diff
@@ -5119,6 +5119,12 @@ Returns the query specified by ID.
     "name": "centos_hosts",
     "description": "",
     "query": "select 1 from os_version where platform = \"centos\";",
+    "team_id": null,
+    "interval": 3600,
+    "platform": "",
+    "min_osquery_version": "",
+    "automations_enabled": true,
+    "logging": "snapshot",
```

Comment:
@zhumo it seems there's a missing field from the current endpoint called `stats` that was never actually documented here. Can we confirm that the plan is to keep that field? The frontend is using it. cc @lucasmrod @juan-fdz-hawa @rachaelshaw @RachelElysia

Reply:
it looks like
<img width="524" alt="Screenshot 2023-07-07 at 4 09 49 PM" src="https://github.com/fleetdm/fleet/assets/61553566/7c3006ac-3598-48a1-8472-738b17a39256">

Reply:
Yes, let's add it!

Reply:
I've just added `stats` to the sample responses.